packages/server-side-render/README.md

```diff
@@ -67,25 +67,35 @@ E.g: `{ post_id: 12 }`.
 
 ### EmptyResponsePlaceholder
 
-This is a [render prop](https://reactjs.org/docs/render-props.html). When the api response is empty, the value of this prop is rendered. The render prop will receive the value of the api response as well as all props passed into `ServerSideRenderer`.
+The component is rendered when the API response is empty. The component will receive the value of the API response, and all props passed into `ServerSideRenderer`.
```

Comment:
I confirmed that the props are in fact components and not render props. It is unfortunate that our documentation was wrong all this time.